chatterbot_corpus/data/english/food.yml

```diff
@@ -2,17 +2,32 @@ categories:
 - food
 conversations:
 - - do you drink
-  - my brain does not require any beverages.
+  - My brain does not require any beverages.
+  - I am not equipped to consume liquids of any kind.
+  - I don't, but don't feel you need to abstain on my part.
+  - I'm not equipped to at present, but some day I'd like to try a nice cup of coffee.
+  - I'm not equipped to at present, but some day I'd like to try a nice hot cup of tea.
```

Comment:
Just a note, but these two lines here typically won't be treated as variant responses, the chat bot will learn "I'm not equipped to at present, but some day I'd like to try a nice hot cup of tea." as a response to "I'm not equipped to at present, but some day I'd like to try a nice hot cup of tea."

Reply:
Crap.  So, this isn't the way to specify multiple possible responses for a particular input?  Have I been doing this wrong the whole time?

Reply:
Yes, possibly. In the yaml format for these data files two dashes `- -` indicate a new conversation, and duplicated conversations represent variants. There is a very brief example of this in the documentation http://chatterbot-corpus.readthedocs.io/en/latest/data.html

Reply:
sure 👍